lib/error.js

```diff
@@ -61,23 +61,24 @@ export const makeError = ({
 	signal = signal === null ? undefined : signal;
 	const signalDescription = signal === undefined ? undefined : signalsByName[signal].description;
 
-	const errorCode = error && error.code;
-
+	const errorCode = error?.code;
 	const prefix = getErrorPrefix({timedOut, timeout, errorCode, signal, signalDescription, exitCode, isCanceled});
 	const execaMessage = `Command ${prefix}: ${command}`;
-	const isError = Object.prototype.toString.call(error) === '[object Error]';
-	const shortMessage = isError ? `${execaMessage}\n${error.message}` : execaMessage;
+	const originalMessage = previousErrors.has(error) ? error.originalMessage : String(error?.message ?? error);
```

Comment:
`String(Symbol())` does not throw, but `${Symbol()}` does. That's a very very tiny edge case, but does not cost much to do.